src/applications/MachineShopSimulator.java

```diff
@@ -11,11 +11,28 @@
     public static final String BAD_MACHINE_NUMBER_OR_TASK_TIME = "bad machine number or task time";
 
     // Data members of MachineShopSimulator
-    static int numMachines; // number of machines
-    static int timeNow; // current time
-    static EventList eList; // pointer to event list
-    static Machine[] machine; // array of machines
-    static int largeTime; // all machines finish before this
+    private static int numMachines; // number of machines

```

Comment:
I think it is good you added these here. But they don't need to be `private`. I would rather see these just be public. Having them be private doesn't really add any security.